orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -340,6 +379,38 @@ private static TypeDescription buildOrcProjectForStructType(Integer fieldId, Typ
     return orcType;
   }
 
+  private static TypeDescription getOrcSchemaForUnionType(Type type, boolean isRequired, Map<Integer, OrcField> mapping,
```

Comment:
nit: Maybe change this `getOrcSchemaForUnionType` method name to `getOrcUnionTypeFromIcebergStructType` ?